cfme/tests/services/test_rest_services.py

```diff
@@ -221,7 +221,6 @@ def unassign_templates(templates):
 
 
 class TestServiceRESTAPI(object):
-    # testing BZ1539741
```

Comment:
I think reference to the BZ could be useful, I'd move it to the docstring of the test (after the description)